x-pack/plugins/license_management/public/sections/license_dashboard/license_status/license_status.js

```diff
@@ -24,19 +24,19 @@ export class LicenseStatus extends React.PureComponent {
     if (isExpired) {
       icon = <EuiIcon color="danger" type="alert" />;
       message = (
-        <span>
+        <span data-test-subj="licenseSubText">
```

Comment:
It might make maintenance easier if we dedupe these three `span` elements by changing them to Fragments like this:

```jsx
    if (isExpired) {
      icon = <EuiIcon color="danger" type="alert" />;
      message = (
        <Fragment>
          Your license expired on <strong>{expiryDate}</strong>
        </Fragment>
      );
      title = `Your ${type} license has expired`;
    } else {
      icon = <EuiIcon color="success" type="checkInCircleFilled" />;
      message = expiryDate ? (
        <Fragment>
          Your license will expire on <strong>{expiryDate}</strong>
        </Fragment>
      ) : (
        <Fragment>
          Your license will never expire.
        </Fragment>
      );
      title = `Your ${type} license is ${status.toLowerCase()}`;
    }
```

And then instantiate the span itself on line 70 like this:

```jsx
    <EuiText color="subdued">
      <span data-test-subj="licenseSubText">
        {message}
      </span>
    </EuiText>
```